src/test/play-3/twirl/uk/gov/hmrc/hmrcfrontend/views/examples/payforatax/bankTransfer.scala.html

```diff
@@ -38,7 +38,7 @@ <h3 class="govuk-heading-m">Account details to use if your account is overseas</
 
 <ul class="govuk-list govuk-list--bullet">
 <li>account number (IBAN) — XXXX XXXX XXXX XXXX XXXX XX</li>
-<li>bank identifier code (BIC) — XXXXXXXX</li>
+<li>Business Identifier Code (BIC) — XXXXXXXX</li>
```

Comment:
Very minor - Does `Business Identifier Code` need to be capitalised? Looks like everything else around it isn't

Reply:
That's how Design Resources have done it in their examples - possibly because BIC is an acronym of the words, whereas the others aren't?
https://github.com/hmrc/design-system/blob/main/src/examples/pay-for-a-tax/bank-transfer/index.njk#L39